src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -982,19 +939,19 @@ void TestCASESession::SessionResumptionStorage(nlTestSuite * inSuite, void * inC
 
     // Create our fabric-scoped node IDs.
     const FabricInfo * fabricInfo = gCommissionerFabrics.FindFabricWithIndex(gCommissionerFabricIndex);
-    NL_TEST_ASSERT(inSuite, fabricInfo != nullptr);
+    EXPECT_NE(fabricInfo, nullptr);
```

Comment:
```suggestion
    ASSERT_NE(fabricInfo, nullptr);
```
Since fabricInfo is about to be dereferenced